### PR TITLE
(#673) Add third party notice to environment documentation

### LINKED ISSUE #673: Add Third Party notices to Chocolatey Environments (Azure, Guide)
## What New Or Updated Would You Like To See?

We need to clearly documents which components in our Quickstart Implementation Guide and Azure Environments are 3rd party software, and document clearly the responsibility of maintaining those applications.

## Why Is It Needed?

From a security perspective it is important that a customer understands which components Chocolatey owns and releases upgrades for, and which components are outside of our area of responsibility. Additionally, it should be clearly documented that these are _not_ SaaS based solutions and once implemented the administration of the environments including software upgrades falls to the customer.

### PR DESCRIPTION
**Pull request by steviecoaster**
## Description Of Changes

Adds documentation around which packages are 3rd party software, and outlines the responsibility of the customer.

## Motivation and Context

From a security perspective we should clearly outline what software Chocolatey Software _does not own_.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

Fixes #673

### CODE REVIEW

File: input/shared/third-party-notice.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
+## Third Party Software Notice
+
+This solution relies on software delivered as a Chocolatey package from the Chocolatey Community Repository. These packages
+have been internalized and made available within the repository included with the solution.
+
+This software includes, but is not limited too:
```
Comment (pauby) [inline input/shared/third-party-notice.txt:6]:
```suggestion
This software includes, but is not limited to:
```

File: input/shared/third-party-notice.txt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- Sonatype Nexus (`nexus-repository`)
+- Jenkins (`jenkins`)
+- Microsoft SQL Server Express (`sql-server-express`)
+- Microsoft SQL Server Management Studio (`sql-server-management-studio`)
+- Dotnet Runtime (`dotnet-6.0-runtime`)
+- Dotnet ASP Net Core Runtime (`dotnet-6.0-aspnetruntime`)
+- Dotnet ASP Net Core Module for IIS (`dotnet-6.0-aspnetcoremodule-v2`)
```
Comment (pauby) [inline input/shared/third-party-notice.txt:14]:
```suggestion
- .NET Runtime (`dotnet-6.0-runtime`)
- .NET ASP Net Core Runtime (`dotnet-6.0-aspnetruntime`)
- .NET ASP Net Core Module for IIS (`dotnet-6.0-aspnetcoremodule-v2`)
```

File: input/shared/third-party-notice.txt [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+### Responsibility
+
+This solution has been provided as a quick way to get up and running with Chocolatey, it is not a SaaS solution. It is the responsibility
```
Comment (pauby) [inline input/shared/third-party-notice.txt:18]:
```suggestion
This solution has been provided as a quick way to get up and running with Chocolatey For Business. It is not a SaaS solution. It is the responsibility
```

File: input/shared/third-party-notice.txt [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+of you, the customer, to ensure the versions of software included as part of the Solution are kept updated per your needs.
```
Comment (pauby) [inline input/shared/third-party-notice.txt:19]:
```suggestion
of you, the customer, to ensure the software included as part of the solution is updated according to your needs.
```

**Review by pauby — CHANGES_REQUESTED** [review]

**Comment by steviecoaster** [comment]
@pauby your comments have been addressed, and are ready for a re-review.

File: input/shared/third-party-notice.txt [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+products that Chocolatey Software maintains. These packages include:
+
+- Chocolatey CLI (`chocolatey`)
+- Chocolatey Licensed Extension (`chocolatey.extension`)
+- Chocolatey Agent (`chocolatey-agent`)
+- Chocolatey GUI (`chocolateygui`)
+- Chocolatey GUI Licensed Extension (`chocolateygui.extension`)
+- Chocolatey Central Management (`chocolatey-management-database`,`chocolatey-management-service`,`chocolatey-management-web`)
+
+If you would like to be subscribed to this mailing list, you can do so by visiting https://groups.google.com/forum/#!forum/chocolatey-announce and providing the email address you would like to use to receive annoucements.
```
Comment (pauby) [inline input/shared/third-party-notice.txt:31]:
```suggestion
Please subscribe to the [mailing list](https://groups.google.com/forum/#!forum/chocolatey-announce) if you would like to receive those annoucements.
```

Comment (pauby) [reply input/shared/third-party-notice.txt:31]: Bare links look terrible 😢

**Review by pauby — CHANGES_REQUESTED** [review]

File: input/shared/third-party-notice.txt (hunk 5 above)
Comment (steviecoaster) [reply input/shared/third-party-notice.txt]: Fixed!

**Comment by pauby** [comment]
@steviecoaster is this PR still needed? Can you see the comment from @ryanrichter94 [above](https://github.com/chocolatey/docs/pull/674#issuecomment-1580868273)?

**Pull request closed without merging**